rocketmq-streams-examples/README.md

```diff
@@ -0,0 +1,67 @@
+## rocketmq-streams-examples
+
+### 1、fileSource example
+```java
+public class FileSourceExample {
+    public static void main(String[] args) {
+        DataStreamSource source = StreamBuilder.dataStream("namespace", "pipeline");
+        source.fromFile("/your/file/path", false)
+                .map(message -> message)
+                .toPrint(1)
+                .start();
+    }
+}
+
+```
+
+
+### 2、rocketmq window example
+```java
+public class RocketmqWindowTest {
+    public static final String NAMESRV_ADDRESS = "127.0.0.1:9876";
+    public static final String RMQ_TOPIC = "NormalTestTopic";
+    public static final String RMQ_CONSUMER_GROUP_NAME = "group-01";
+    public static final String TAGS = "*";
+
+    /**
+     * 1、before run this case, make sure some data has already been rocketmq.
+     * 2、rocketmq allow create topic automatically.
+     */
+    public static void main(String[] args) {
+        DataStreamSource source = StreamBuilder.dataStream("namespace", "pipeline");
+
+        source.fromRocketmq(
+                RMQ_TOPIC,
```

Comment:
It will be better if you can provide the method of how to send messages on this topic and the format of this message.